astroplan/constraints.py

```diff
@@ -546,6 +546,48 @@ def __init__(self, min=None, max=None, ephemeris=None):
         self.max = max
         self.ephemeris = ephemeris
 
+    @classmethod
+    def dark(cls, **kwargs):
+        """
+        This will use default values (acceptable fractional illumination)
+        of max=0.25 with no minimum. You can input min= or max= to change
+        the values and ephemeris= if a different ephemeris is desired,
+        by default it uses ``astropy.coordinates.solar_system_ephemeris.set``
```

Comment:
Leave the ``.set`` ending out of it, there is no API link for that, and also it's irrelevant I think.